### PR TITLE
Allow creating jar for Windows too

### LINKED ISSUE #722: jar doesn't exist
After this PR: https://github.com/gluonhq/substrate/pull/690 was integrated, as a full path, we need to set `jar.exe

### PR DESCRIPTION
**Pull request by jperedadnr**
Fixes #722

### CODE REVIEW

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/model/ClassPath.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5929475&bid=19428350)

File: src/main/java/com/gluonhq/substrate/model/Triplet.java [hunk 1]
```diff
@@ -58,6 +58,27 @@ public static Triplet fromCurrentOS() throws IllegalArgumentException {
         }
     }
 
+    /**
+     * @return true if host is Windows
+     */
+    public static boolean isWindowsHost() {
+        return new Triplet(Constants.Profile.WINDOWS).equals(fromCurrentOS());
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/model/Triplet.java:65]: does this really requires the creation of a new Object?

Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/model/Triplet.java:65]: I could simply do:
```
String osName  = System.getProperty("os.name").toLowerCase(Locale.ROOT);
return osName.contains("windows");
```
instead?
Then we could also revert `fromCurrentOS` to use these `isXHost` methods?

Comment (johanvos) [reply src/main/java/com/gluonhq/substrate/model/Triplet.java:65]: That sounds better indeed.

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/model/ClassPath.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5929475&bid=19428350)

File: src/main/java/com/gluonhq/substrate/model/Triplet.java (hunk 1 above)
Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/model/Triplet.java]: ok, pushed

File: src/main/java/com/gluonhq/substrate/model/Triplet.java [hunk 2]
```diff
@@ -35,6 +35,8 @@
 
 public class Triplet {
 
+    private static final String OS_NAME  = System.getProperty("os.name").toLowerCase(Locale.ROOT);
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/model/Triplet.java:38]: Can you add a comment here that makes it clear this is evaluated at build-time?

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/model/ClassPath.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5929475&bid=19428350)

File: src/main/java/com/gluonhq/substrate/model/Triplet.java (hunk 2 above)
Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/model/Triplet.java:41]: Added

**Pull request merged**